source/III.VC.CLEO/CustomOpcodes.cpp

```diff
@@ -2014,17 +2014,16 @@ eOpcodeResult CustomOpcodes::OPCODE_0ADA(CScript *script)
 eOpcodeResult CustomOpcodes::OPCODE_0ADB(CScript *script)
 {
 	script->Collect(2);
-	unsigned mi = game.Scripts.Params[0].nVar;
```

Comment:
`mi` is actually a quite common abbreviation in Rockstar source code. Stands for `model index`

Reply:
I find excessive abbreviations confusing.  First,not everyday knows what `mi` means, secondly it is harder to spot them in action.

I can revert this if necessary.

Reply:
you can keep it, it was just a remark